src/lib/api/api.go

```diff
@@ -467,6 +468,52 @@ func (a *API) ModifyIntervalDuration(w http.ResponseWriter, r *http.Request) {
 	a.send(w, a.marshalled())
 }
 
+// Edit or remove a route from the config screen.
+func (a *API) ModifyConfigRoutes(w http.ResponseWriter, r *http.Request) {
+	if err := r.ParseForm(); err != nil {
+		panic(err)
+	}
+	var req struct {
+		Action      string `json:"action"`
+		NewRoute    string `json:"newRoute"`
+		ActualRoute string `json:"actualRoute"`
+	}
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&req); err != nil {
+		panic(err)
+	}
+	switch req.Action {
+	case "edit-route":
```

Comment:
since `route` is expected to be redundant here, how about simply `edit`, `delete`, etc.?